### PR TITLE
docs: initial emojimon v2 tutorial

### PR DESCRIPTION
**Pull request by nickcernera**

### CODE REVIEW

File: docs/pages/_meta.js [hunk 1]
```diff
@@ -36,4 +36,5 @@ export default {
     newWindow: true,
   },
   contribute: "Contribute",
+  "preface-the-ecs-model": "Preface: the ECS model",
```
Comment (holic) [inline docs/pages/_meta.js:39]:
```suggestion
```

File: docs/pages/emojimon-tutorial/0.-getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Getting started
```
[the remark this answers is not in the document]
Comment (nickcernera) [reply docs/pages/emojimon-tutorial/0.-getting-started.mdx:1]: oh yep makes sense

File: docs/pages/tutorial/emojimon/deploy-to-testnet.mdx [hunk 3]
```diff
@@ -0,0 +1 @@
+# Deploy to testnet
```
Comment (holic) [inline docs/pages/tutorial/emojimon/deploy-to-testnet.mdx:1]: testnet instructions haven't changed much, wondering if we can just copy that over from the old tutorial

File: docs/pages/tutorial/emojimon.mdx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Introduction
+
+By the end of this tutorial you will learn how to create your own EVM application, a pokémon-inspired onchain game called Emojimon, using [MUD](https://v2.mud.dev/). This will help you understand how MUD works and how it can be used to build any other EVM application or onchain game.
```
Comment (ludns) [inline docs/pages/tutorial/emojimon.mdx:3]: all the links are absolute, and some point to V2.
Can you make links relative?
eg: `https://mud.dev/client-side#recs-the-reactive-ecs-database` -> `/client-side#recs-the-reactive-ecs-database`

**Pull request merged**